index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header class="Top">
+            <nav class="menu">
+                <img class="logo" src="images/toolbar-mobile.png" alt=""><a href="#"></a>
+            </nav>
+         </header>
```

Comment:
As required by the previous reviewer, you have to split your image in 2 images, and update your code like this

```
 <header>
        <a href=""> <img src="Mylogo" ></a>
        <nav>
         < img src="hamburger.png">
        </nav>
      </header>
```